Examples compilation error

ispc-v1.12.0-windows packaged example compilation failed.

``` powershell
> cmake --build . --target .\aobench
[1/2] Building CXX object aobench\CMakeFiles\aobench.dir\ao.cpp.obj
FAILED: aobench/CMakeFiles/aobench.dir/ao.cpp.obj
C:\PROGRA~2\MICROS~1\2019\COMMUN~1\VC\Tools\MSVC\1423~1.281\bin\Hostx64\x64\cl.exe  /nologo /TP  -Iaobench /fp:fast /Oi /showIncludes /Foaobench\CMakeFiles\aobench.dir\ao.cpp.obj /Fdaobench\CMakeFiles\aobench.dir\ /FS -c C:\Users\cloud\Desktop\ispc-v1.12.0-windows\examples\aobench\ao.cpp
C:\Users\cloud\Desktop\ispc-v1.12.0-windows\examples\aobench\../timing.h(75): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
C:\Users\cloud\Desktop\ispc-v1.12.0-windows\examples\aobench\../timing.h(75): error C2146: syntax error: missing ';' before identifier 'uint64_t'
C:\Users\cloud\Desktop\ispc-v1.12.0-windows\examples\aobench\../timing.h(75): error C2143: syntax error: missing ';' before '{'
C:\Users\cloud\Desktop\ispc-v1.12.0-windows\examples\aobench\../timing.h(75): error C2447: '{': missing function header (old-style formal list?)
C:\Users\cloud\Desktop\ispc-v1.12.0-windows\examples\aobench\../timing.h(86): fatal error C1083: Cannot open include file: 'sys/time.h': No such file or directory
ninja: build stopped: subcommand failed.
```

--- a/examples/CMakeLists.txt
+++ b/examples/CMakeLists.txt
@@ -40,6 +40,11 @@ project(${PROJECT_NAME})
 
 include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/AddISPCExample.cmake)
 
+if(WIN32)
+    add_compile_definitions(WIN32)
+    add_compile_options(/EHsc)
+endif()
+
 if (ISPC_BUILD)
     set (ISPC_EXECUTABLE $<TARGET_FILE:ispc>)
     set (EXAMPLES_ROOT ${CMAKE_SOURCE_DIR}/examples)

